pep/app/pods/components/keyboard-shortcuts/template.hbs

```diff
@@ -0,0 +1 @@
+<div class="shortcut-element" ...attributes {{did-insert this.setupShortcuts}}></div>
```

Comment:
this feels off/non-sematic to me, to have an empty div just to setup keyboard shortcut bindings. I think this is exactly where you would want to create a custom modifier, and then use that on the root element of the applicable templates, or w/e part of it makes sense. e.g.

```hbs
<div class="application-container" {{register-shortcuts this.shortcuts}}>
  ...
</div>
```

```hbs
<nav class="navbar navbar-expand fixed-top navbar-dark bg-gradient-primary" {{register-shortcuts this.shortcuts}} ...attributes>
   ...
</nav>
```
etc.